sentinel/src/main/AndroidManifest.xml

```diff
@@ -112,13 +112,7 @@
             <meta-data
                 android:name="com.infinum.sentinel.SentinelInitializer"
                 android:value="androidx.startup" />
-        </provider>
 
-        <provider
-            android:name="androidx.startup.InitializationProvider"
-            android:authorities="${applicationId}.androidx-startup"
-            android:exported="false"
-            tools:node="merge">
```

Comment:
Why is Sentinel removing the work manager initializer? Does it even use anything from the work manager?

I'm worried about this because if the app add sentinel they would "hiddenly" remove the initialization of the work manager. Everything would be fine for the apps using [custom work manager initialization](https://developer.android.com/develop/background-work/background-tasks/persistent/configuration/custom-configuration#remove-default), but for the apps using the default one, work manager wouldn't be initialized anymore.

I'd be calmer if we would remove this code completely, therefore not removing the work manager initialization, and therefore leaving the work manager setup as it is defined in the apps.

Reply:
I will look into that option as well. It should work.

Reply:
Just to double check. You are suggesting to remove this whole part?
```
<meta-data
                android:name="androidx.work.WorkManagerInitializer"
                android:value="androidx.startup"
                tools:node="remove" />
```

Do you then think we should add this to our README as a reminder for users that they will have to do it in their projects?
I will leave the decision to Karlo as he is the owner.

Reply:
Yes, I'm suggesting that, but not in case users would need to add this to their projects only because they have Sentinel.
I'm suggesting removing this if it is redundant.
The only thing when users should add this is in case they use a work manager in their project.

Does Sentinel even use a work manager? What for?

Reply:
I think only usage is in `SentinelWorkManager` class. It is used for certificates checking. Interestingly enough, this is the second time certificates causing problems 😄 .

Reply:
And I think that is a problem. Since WorkManager can be initialized only once (next time it throws), any application using Work Manager would fail to supply their own workers and initialize the WorkManager. That means, the application dev has to choose, either to use WorkManager and no Sentinel, or vice-versa.

That's the problem we should investigate further and see how to avoid the clash.